deviceplugin/usb.go

```diff
@@ -86,32 +109,32 @@ func queryUSBDeviceCharacteristicsByDirectory(dir os.DirEntry) (result *usbDevic
 		// There shouldn't be any raw files in this directory, but just in case.
 		return result, fmt.Errorf("not a directory")
 	}
+	fqPath := usbDevicesDir + dir.Name()
```

Comment:
Let's use `filepath.Join` just for extra safety and correctness

https://pkg.go.dev/path/filepath#Join

Reply:
TIL `filepath.Join()` is a thing that exists, I can sleep soundly now 378ce05c